### PR TITLE
Topic/863/admin survey details

### PR DESCRIPTION
**Pull request by steventlamb**

### CODE REVIEW

**Comment by steventlamb** [comment]
screenshot from 2015-03-13 16 52 03
screenshot from 2015-03-13 16 52 14
screenshot from 2015-03-13 16 52 26

**Comment by steventlamb** [comment]
Assigning to @jwalgran for review because @RickMohr and @maurizi were part of the discussion so have already seen this.

File: src/nyc_trees/apps/survey/routes.py [hunk 1]
```diff
@@ -70,6 +70,11 @@
     route(GET=do(render_template('survey/survey_detail.html'),
                  v.survey_detail)))
 
+confirm_survey = individual_mapper_do(
+    route(GET=do(render_template('survey/survey_detail.html'),
+                 update_with({'show_controls': True}),
```
Comment (jwalgran) [inline src/nyc_trees/apps/survey/routes.py]: I like this "middleware" for situationally adding to the context.

**Comment by jwalgran** [comment]
+1 works perfectly. I verified that a normal survey still shows the option buttons on the review page.

**Pull request merged**